1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,4 +1,17 @@
-function setAlarm() {}
+function setAlarm() {
+  let inputEl = document.querySelector("#alarmSet");
+  let timer = inputEl.value;
+  let timeRemainEl = document.querySelector("#timeRemaining");
+  timeRemainEl.innerText = "Time Remaining: 00:" + timer;
+  const myInterval = setInterval(() => {
+    timer--;
+    timeRemainEl.innerText = "Time Remaining: 00:" + timer;
+    if (timer === 0) {
+      clearInterval(myInterval);
+      playAlarm();
+    }
```

Comment:
Have you used the pause function anywhere? what can we do using the stop alarm button?